junit-launcher/src/main/java/org/junit/gen5/launcher/main/Root.java

```diff
@@ -44,15 +50,13 @@ TestDescriptor getTestDescriptorFor(TestEngine testEngine) {
 	}
 
 	void applyPostDiscoveryFilters(TestDiscoveryRequest discoveryRequest) {
-		TestDescriptor.Visitor filteringVisitor = (descriptor, remove) -> {
-			if (!descriptor.isTest())
-				return;
-
-			if (isExcluded(discoveryRequest, descriptor)) {
+		Filter<TestDescriptor> postDiscoveryFilter = composeFilters(discoveryRequest.getPostDiscoveryFilters());
```

Comment:
AFAIK the correct formatting is checked by the build. No manual work needed to check this. Am I right, @sbrannen?